SuperSquareBros.cpp

```diff
@@ -1765,7 +1765,7 @@ class Entity {
             y += yVel * dt;
 
             // Here check collisions...
-            for (uint16_t i = 0; i < foreground.size(); i++) {
+            for (size_t i = 0; i < foreground.size(); i++) {
```

Comment:
I've been moving to @Gadgetoid 's suggestion of using `for (Tile& tile : foreground) ` etc, I must have missed these.

Reply:
Convenient for me, as I didn't get a rebase conflict :smile:

(I could remove that bit/switch it to range-based though)